src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsCompactChat.kt

```diff
@@ -57,7 +60,9 @@ object HoppityEggsCompactChat {
 
         return if (duplicate) {
             val format = lastDuplicateAmount?.let { NumberUtil.format(it) } ?: "?"
-            "$mealName Egg! §7Duplicate $lastName §7(§6+$format Chocolate§7)"
+            val timeHumanized = lastDuplicateAmount?.let { ChocolateFactoryAPI.timeUntilNeed(it).format(maxUnits = 2) } ?: "?"
```

Comment:
```suggestion
            val timeHumanized = lastDuplicateAmount?.let {
                ChocolateFactoryAPI.timeUntilNeed(it).format(maxUnits = 2)
            } ?: "?"

```

I feel this makes the code slightly more readable, but up to you/another reviewer